tests/lib/rules/yoda.js

```diff
@@ -992,6 +1017,110 @@ ruleTester.run("yoda", rule, {
                     type: "BinaryExpression"
                 }
             ]
+        },
+        {
+            code: "if (`green` < x.y && x.y < `blue`) {}",
```

Comment:
this is invalid as `"green"` is greater than `"blue"`

![image](https://user-images.githubusercontent.com/26347874/77098328-3221a500-6a38-11ea-8728-27384599f9f0.png)

Reply:
:+1: